<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriorityWidget</class>
 <widget class="QWidget" name="PriorityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="lightlyButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 250, 150);</string>
     </property>
     <property name="text">
      <string>Lightly</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="importantButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="text">
      <string>Important</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="veryImportantButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 150, 150);</string>
     </property>
     <property name="text">
      <string>VeryImportant</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
